buildstockbatch/aws/aws.py

```diff
@@ -1525,15 +1040,88 @@ def container_repo(self):
             repo = resp["repository"]
         return repo
 
+    @property
+    def image_url(self):
+        return f"{self.container_repo['repositoryUri']}:{self.job_identifier}"
+
     def build_image(self):
```

Comment:
`build_image()` should be safe to move into DockerBase, if you want to make that change here. (Or I can move it in a separate PR, if that's easier.)

We're using the same code in the GCP branch, other than renaming the `.aws_docker_image` dir. (Maybe that could just be something like `.cloud_docker_image` instead?)

Reply:
Let's do this in a separate PR.